README.md

```diff
@@ -10,6 +10,7 @@ This project provides a collective list of Generative AI (GenAI) examples such a
 ---
 
 <div align="left">
+This project provides a collective list of Generative AI (GenAI) and RAG (Retrieval and Augmentation) examples such as chatbot with question and answering (ChatQnA), code generation, document summary, etc. The examples are developed by leveraging the ecosystem components such as LangChain with Intel optimizations, therefore delivering the high performance for your deployment on Intel platforms.
```

Comment:
RAG (Retrieval and Augmentation) should be RAG (Retrieval Augmentation Generation)